src/term/flavour_py/mod.rs

```diff
@@ -0,0 +1,98 @@
+mod order_enums;
+pub mod to_lang;
+
+use indexmap::IndexMap;
+use interner::global::GlobalString;
+
+use crate::term::Name;
+
+use super::Op;
+
+#[derive(Clone, Debug)]
+pub enum Term {
+  // "None"
+  None,
+  // [a-zA-Z_]+
+  Var { nam: Name },
+  // [0-9_]+
+  Num { val: u32 },
+  // {fun}(args,)
+  Call { fun: Box<Term>, args: Vec<Term> },
+  // "fun" {pat} -> {bod}
+  Lam { pat: AssignPattern, bod: Stmt },
+  // {nam} "{" {fields} "}"
+  Enum { nam: Name, fields: Vec<(Name, Term)> },
+  // {lhs} {op} {rhs}
+  Bin { op: Op, lhs: Box<Term>, rhs: Box<Term> },
+  // "\"" ... "\""
+  Str { val: GlobalString },
+  // "[" ... "]"
+  Lst { els: Vec<Term> },
+  // "(" ... ")"
+  Tup { els: Vec<Term> },
+}
+
+#[derive(Clone, Debug)]
+pub struct MatchArm {
+  pub lft: Option<Name>,
+  pub rgt: Stmt,
+}
+
+#[derive(Clone, Debug)]
+pub enum AssignPattern {
+  // [a-zA-Z_]+
+  Var(Name),
+  // "(" ... ")"
+  Tup(Vec<Name>),
+}
+
+#[derive(Clone, Debug)]
+pub enum Stmt {
+  // {pat} = {val} ";" {nxt}
+  Assign { pat: AssignPattern, val: Box<Term>, nxt: Box<Stmt> },
+  // "if" {cond} ":"
+  //  {then}
+  // "else" ":"
+  //  {otherwise}
+  If { cond: Box<Term>, then: Box<Stmt>, otherwise: Box<Stmt> },
+  // "match" {arg} ":"
+  //   case {lft} ":" {rgt}
+  Match { arg: Box<Term>, bind: Option<Name>, arms: Vec<MatchArm> },
+  // "return" {expr} ";"
+  Return { term: Box<Term> },
+}
+
+// Name "{" fields, "}"
+#[derive(Clone, Debug)]
+pub struct Variant {
+  pub name: Name,
+  pub fields: Vec<Name>,
+}
+
+// "def" {name} "(" {params} ")" ":" {body}
+#[derive(Clone, Debug)]
+pub struct Definition {
+  pub name: Name,
+  pub params: Vec<Name>,
+  pub body: Stmt,
+}
+
+// "enum" "{" {variants} "}"
```

Comment:
I think we should change the enum syntax to be indentation based
```
enum MyEnum:
  Ctr1(a, b, c)
  Ctr2
  Ctr3(a, b, c, d)
```

Reply:
It is already based on the indentation, I just forgot to update this comment after implemented it